cmake/SndFileConfig.cmake.in

```diff
@@ -5,19 +5,24 @@ set(SndFile_VERSION_PATCH @PROJECT_VERSION_PATCH@)
 
 set (SndFile_WITH_EXTERNAL_LIBS @SndFile_WITH_EXTERNAL_LIBS@)
 set (SndFile_WITH_MPEG @SndFile_WITH_MPEG@)
+set (SndFile_STATIC NOT @BUILD_SHARED_LIBS@)
```

Comment:
```suggestion
set (SndFile_IS_SHARED @BUILD_SHARED_LIBS@)
```

This code does not do what you expect, you're actually building a list of `NOT;<value of BUILD_SHARED_LIBS>` which will always evaluate to true.

For example:
```cmake
set(FOO ON)
set(BAR NOT ${FOO})
if(BAR)
  message("BAR was evaluated to true with the value: '${BAR}'")
else()
  message("BAR was evaluated to false with the value: '${BAR}'")
endif()
```
Prints:
```
 BAR was evaluated to true with the value: 'NOT;ON'
```

Reply:
You are absolutely right, CMake does not evaluate boolean expressions except within if() and while().

Unfortunately, I don't have time right now to work on this issue. I've added you to my fork as collaborator, so you probably can push more commits to this PR if you want.  If you don't, I think that will close this PR in a few days and delete my fork.

In any case, I don't have confidence in this project anymore. This PR has been held without comments for a month already, but you can see that they have held others during years. Good luck!

Reply:
I appreciate the offer, but I went ahead an opened a new PR (#925) since I felt it was easier to start fresh compared to partially reverting changes.

I hope you understand!